### DISCUSSION TITLE
I accidentally added github student to my old github account

### DISCUSSION THREAD

**Discussion by 0x-s13i** [community/community#43197] [GitHub Education]
Hi,

I would like to use an email address that was already registered to another account to apply for a student account. Both accounts are mine, and I have removed the email address from the first account.

I was accepted for a GitHub student account on my old GitHub account (email: ***@gmail.com) - Happy to share in a DM

I realised that the account I'm currently messaging from (email: ***@googlemail.com - note the difference in @gmail and @googlemail) is actually the one I use.

I removed my student email address from my old account and added it to this account (emails: ***@googlemail.com / ***@kellogg.ox.ac.uk)

I tried to reapply for a GitHub student account with this account. I thought that removing my email address from my old account would be enough and that I would be able to reapply.

Now, I am unable to reapply. Would it be possible for me to reapply using this account? I would much prefer to use this account, and this is the account I currently develop with.

The university I am studying at is Kellog College, University of Oxford.
I would be using my GitHub student account to:
- complete my assignments
- learn new languages and skills
- research my software engineering masters
- work on personal projects.

I can attach an image with my student card as proof of my enrolment.

**Comment by jfullstackdev** [comment]
posting it here might help 

and Support might help too, although there is no assurance

https://support.github.com/contact/account

**Reply by 0x-s13i** [reply]
I tried that before, and 40 days after opening my ticket, they closed it and told me to post here for help 😅

**Reply by jfullstackdev** [reply]
yup, and for the Education Pack, I'm really seeing so many failed requests, even here
